rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -377,6 +377,39 @@ def _bind_ports(self, lports, sandboxes, port_bind_args):
         if wait_up:
             self._wait_up_port(lports, wait_sync, install_method)
 
+    @atomic.action_timer("ovn_network.bind_overlay_ports")
+    def _bind_overlay_ports(self, farms, port_bind_args):
```

Comment:
this one seems redundant with _bind_ports. It is better to reuse _bind_ports and update the function for the case where number of sandboxes are bigger than number of ports to bind.